src/styles/blocks/header.scss

```diff
@@ -0,0 +1,10 @@
+.header {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
```

Comment:
The header should also have a bolder font-weight and an ::after on Apple:
![image](https://github.com/mate-academy/layout_catalog/assets/67425140/84819a4d-5dd5-4f08-b162-2f3e60ef1568)